gatsby-node.js

```diff
@@ -5,11 +5,18 @@
  */
 
 // You can delete this file if you're not using it
+const path = require("path")
 
 exports.onCreateWebpackConfig = ({ actions, loaders, plugins, stage }) => {
   actions.setWebpackConfig({
     resolve: {
-      modules: ['src', 'node_modules']
-    }
+      modules: ['src', 'node_modules'],
+      alias: {
+        "_variables.sass": path.resolve(
+          __dirname,
+            "./src/styles/_variables.sass"
```

Comment:
Please make sure that this actually works.
It has to properly override the default variables and preserve the non-overridden default variables.
Thanks.

Reply:
You can try it by changing its primary color variable and importing a component from `react-bulma-components` (e.g. Hero component), then render it in our `Header` component.
See whether it works as expected or not.

Reply:
This is when we change a component and use `bulma` components.
Note: this is screenshot just for test (make sure integration with react-bulma-components)
![image](https://user-images.githubusercontent.com/16365952/56400663-40db6700-627f-11e9-8569-841513a58e38.png)

Reply:
For `@emotion/core` integration. 👍  
![image](https://user-images.githubusercontent.com/16365952/56400947-b136b800-6280-11e9-98f7-caf61c4d552f.png)